frontend/src/components/SimpleMenuActions.tsx

```diff
@@ -1,56 +1,69 @@
 import * as React from 'react';
-import {
-  Dropdown,
-  DropdownItem,
-  DropdownSeparator,
-  KebabToggle,
-} from '@patternfly/react-core/deprecated';
+import { MenuToggle, Dropdown, DropdownItem, Divider, DropdownList } from '@patternfly/react-core';
+import { EllipsisVIcon } from '@patternfly/react-icons';
 
 type Item = {
   key: string;
   label: React.ReactNode;
   onClick: (key: string) => void;
   isDisabled?: boolean;
-  tooltip?: React.ReactNode;
+  tooltip?: {
+    content: React.ReactNode;
+  };
 };
 type Spacer = { isSpacer: true };
 const isSpacer = (v: Item | Spacer): v is Spacer => 'isSpacer' in v;
 
 type SimpleDropdownProps = {
   dropdownItems: (Item | Spacer)[];
+  testId?: string;
 } & Omit<
   React.ComponentProps<typeof Dropdown>,
-  'isOpen' | 'isPlain' | 'position' | 'toggle' | 'dropdownItems' | 'onChange'
+  'isOpen' | 'isPlain' | 'popperProps' | 'toggle' | 'onChange'
 >;
 
-const SimpleMenuActions: React.FC<SimpleDropdownProps> = ({ dropdownItems, ...props }) => {
+const SimpleMenuActions: React.FC<SimpleDropdownProps> = ({ dropdownItems, testId, ...props }) => {
   const [open, setOpen] = React.useState(false);
 
   return (
     <Dropdown
       {...props}
       isOpen={open}
-      isPlain
-      toggle={<KebabToggle onToggle={() => setOpen(!open)} />}
-      position="right"
-      dropdownItems={dropdownItems.map((itemOrSpacer, i) =>
-        isSpacer(itemOrSpacer) ? (
-          <DropdownSeparator key={`spacer-${i}`} />
-        ) : (
-          <DropdownItem
-            key={itemOrSpacer.key}
-            isDisabled={itemOrSpacer.isDisabled}
-            tooltip={itemOrSpacer.tooltip}
-            onClick={() => {
-              itemOrSpacer.onClick(itemOrSpacer.key);
-              setOpen(false);
-            }}
-          >
-            {itemOrSpacer.label}
-          </DropdownItem>
-        ),
+      onOpenChange={(isOpened) => setOpen(isOpened)}
+      toggle={(toggleRef) => (
+        <MenuToggle
+          aria-label="Simple menu actions"
```

Comment:
Tests needed to change because of this label. To be consistent, keep the old value.
```suggestion
          aria-label="Actions"
```